pyproject.toml

```diff
@@ -43,6 +43,8 @@ keywords = [
 dynamic = ["version"]
 dependencies = [
     "elastic-transport>=8.15.1,<9",
+    "python-dateutil",
```

Comment:
For what it's worth, this one makes me a bit sad, as it still pulls six to this day: https://inspector.pypi.io/project/python-dateutil/2.9.0.post0/packages/ec/57/56b9bcc3c9c6a792fcbaf139543cee77261f3651ca9da0c93f5c1221264b/python_dateutil-2.9.0.post0-py2.py3-none-any.whl/python_dateutil-2.9.0.post0.dist-info/METADATA

Reply:
Let me see if we can drop it.